server/routes/serviceProviderReferrals/serviceProviderReferralsController.test.ts

```diff
@@ -699,7 +699,7 @@ describe('POST /service-provider/action-plan/:actionPlanId/appointment/:sessionN
         .type('form')
         .send({
           attended: 'yes',
-          additionalAttendanceInformation: 'Alex made the session on time',
+          'additional-attendance-information': 'Alex made the session on time',
```

Comment:
Do you also need to change this in the other tests here, and the `name` attribute on the form input?

Reply:
(and I guess also in the presenter, when pulling the values back out from the user input data?)

Reply:
If you want to rename the parameter it might be best doing it as a separate commit.

Reply:
Have extracted this out into a separate commit.